### PR TITLE
WIP: migration

### PR DESCRIPTION
**Pull request by edgesoft**
**WIP**

Is this something that we are aiming for or do you have other suggestions? Will do the same for Signals, Services -> Providers etc.

### CODE REVIEW

**Comment by edgesoft** [comment]
Just need some pointers about this.

File: content/migration/01_start_migration.en.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+```js
+import {Controller} from 'cerebral'
+
+import Home from './modules/Home'
+import Admin from './modules/Admin'
+
+const controller = Controller({
+  modules: {
+    home: Home,
```
Comment (gaspard) [inline content/migration/01_start_migration.en.md]: I think it should be: `home: Home(),`

Comment (edgesoft) [reply content/migration/01_start_migration.en.md]: Yes, Will change. Might depend on export default {state:{}, signals{} } or export default (module) => ...

Comment (gaspard) [reply content/migration/01_start_migration.en.md]: Yes you are right. Might be nice to see what will be the most common usage for 2.x.

File: content/migration/01_start_migration.en.md [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+Now instead of having services in the context object in an action you would have the **axios** object.
+
+```js
+function myAction({axios}) {
+
+}
+```
+
+### Views
+How you import view specific packages has changed due to that cerebral now is a monorepo. In 1.x you would import Container like this.
```
Comment (Guria) [inline content/migration/01_start_migration.en.md]: In fact monorepo doesn't plays a role in import path. We just decided to make it part of core package.

Comment (edgesoft) [reply content/migration/01_start_migration.en.md]: Ok! Will fix that!

Comment (edgesoft) [reply content/migration/01_start_migration.en.md]: #b70f8d8

**Comment by Guria** [comment]
Can also note that we don't need `immediate` signal flag anymore

**Comment by edgesoft** [comment]
Will make a note on that @Guria

**Review by Guria — APPROVED** [review]
Very good page to start. I will merge it as is for now.
Fell free to add more edits to migration guilde if any needed.

**Pull request merged**